testnet.toml

```diff
@@ -1,11 +1,11 @@
 [parity]
 chain = "ropsten"
-base_path = "/data/ethereum"
-db_path = "/data/ethereum/chains"
-keys_path = "/data/ethereum/keys"
+base_path = "./testnet"
```

Comment:
This file was being used by `k8s/testnet.yaml` for deployment to google cloud. I've since made another copy called `k8s/ropsten.toml` to house settings specific to k8s and leave this one dedicated to local docker runs.